src/api/package/assemblies.ts

```diff
@@ -63,7 +63,7 @@ async function fetchAssembly(
     version = version.substring(1, version.length);
   }
 
-  const assemblyPath = `${getAssetsPath(name, version, scope)}/jsii.json`;
+  const assemblyPath = `${getAssetsPath(name, version, scope)}/assembly.json`;
```

Comment:
https://github.com/cdklabs/construct-hub/blob/main/src/backend/shared/constants.lambda-shared.ts#L27

Reply:
Could you update the corresponding service worker route as well? You can search within `service-worker.ts` for `jsii.json`

Reply:
Good catch

Reply:
Added